server/src/step/step.controller.js

```diff
@@ -0,0 +1,13 @@
+import { Step } from '@/shared/database/index';
+
+const newStep = async (req, res, next) => {
+  try {
+    const step = await Step.create(req.body);
+
+    res.json({ stepId: step.id });
+  } catch (err) {
+    next(new Error());
+  }
+};
+
+module.exports = { newStep };
```

Comment:
we use es6 modules so please start using `export { smn }` instead of `module.exports = { smn }`